src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.cs

```diff
@@ -57,8 +57,9 @@ public static partial class PlatformDetection
         public static bool SupportsAlpn => (IsWindows && !IsWindows7) ||
             ((!IsOSX && !IsWindows) &&
             (OpenSslVersion.Major >= 1 && (OpenSslVersion.Minor >= 1 || OpenSslVersion.Build >= 2)));
-        public static bool SupportsClientAlpn => SupportsAlpn ||
-            (RuntimeInformation.IsOSPlatform(OSPlatform.OSX) && PlatformDetection.OSXVersion > new Version(10, 12));
+        public static bool SupportsClientAlpn => SupportsAlpn || (IsOSX && PlatformDetection.OSXVersion > new Version(10, 12));
+        // OpenSSL 1.1.1 and above.
+        public static bool SupportsTls13 => !IsWindows && !IsOSX && (OpenSslVersion.Major >= 1 && OpenSslVersion.Minor >= 1 && OpenSslVersion.Build >= 1);
```

Comment:
The version check here is incorrect.  The next version will be 3.0(.0?), so Major will be >= 1, but Minor will be 0.